Makefile.include

```diff
@@ -123,7 +123,7 @@ endif
 WERROR ?= 1
 export WERROR
 ifeq ($(WERROR),1)
-  CFLAGS += -Wall -Werror -Wextra
+  CFLAGS += -Wall -Werror -Wextra -Werror=implicit-fallthrough=2
```

Comment:
Why do we need this? Are some comments wrongly formatted to match the gcc default regex?

Reply:
Not wrongly formatted, but default is `-Werror=implicit-fallthrough=3`, which only allows (or matches) on one-line comments with very limited (and case sensitive) wording, e.g., `/* falls through */`, but some like to add an explanation like `/* falls through intentionally because of XYZ */`, see [here](https://github.com/RIOT-OS/RIOT/blob/master/drivers/slipdev/slipdev.c#L162) for instance, which does not match and hence causes an error.

However, considering 17000++ errors this gives us for older compilers that do not recognise this option - we better fix/adapt the comments - I guess.